src/app/(admin)/chats/_components/ChatManagementContent.tsx

```
@@ -1,9 +1,15 @@
 'use client';
 
-import { useCallback } from 'react';
+import { useCallback, useMemo } from 'react';
+import { useSearchParams } from 'next/navigation';
```

ESLint

[error] `next/navigation` import should occur before import of `react`

(import/order)